models/aeif_cond_beta_multisynapse.h

```diff
@@ -454,9 +500,38 @@ aeif_cond_beta_multisynapse::set_status( const DictionaryDatum& d )
   // consistent.
   Archiving_Node::set_status( d );
 
+  /**
```

Comment:
This `set_status()` is so large it should not be inlined and should be moved to the cpp-file.

Reply:
Done in eb41277